Doc/library/dis.rst

```diff
@@ -398,13 +398,18 @@ The Python compiler currently generates the following bytecode instructions.
    Push the *i*-th item to the top of the stack. The item is not removed from its
    original location.
 
+   The stack is indexed from 1, so COPY 1 will copy TOS.
+
    .. versionadded:: 3.11
 
 
 .. opcode:: SWAP (i)
 
    Swap TOS with the item at position *i*.
 
+   The stack is indexed from 1, so SWAP 2 will swap the two element at the top
+   of the stack.
+
```

Comment:
Perhaps the explanation of TOS etc under the "**Binary and in-place operations**" heading should be moved up?

Reply:
I considered doing so. But since it was a bigger change I though I would wait to see what reviewers thought. 

I will move it as soon as I get a chance since we agree it makes sense.